### ISSUE TITLE
Update range expansion to group ranges against a common field

### ISSUE THREAD

**Issue by ivakegg** [NationalSecurityAgency/datawave#873]
In the default query planner when we expand bounded ranges we create a single scan per range.  Instead we could group multiple scans against the same field together and use next/seek to move between the ranges.  The hope is that this will decrease our time to the first page of results for queries with many ranges against a single field.

**Comment by ivakegg** [comment]
Now that we do bounded range lookups in parallel, this is no longer needed

**Comment by ivakegg** [comment]
Now that we do bounded range lookups in parallel, this is no longer needed
